tests/helper.py

```diff
@@ -2,6 +2,195 @@
 Helper functions for unit tests.
 """
 
+import unittest
+
+import geomstats.backend as gs
+
+
+class TestGeomstatsMethods(unittest.TestCase):
+    def assertScalar(self, result, n_samples=1):
+        return self.assertTrue(
+            gs.allclose(result.shape, (n_samples, 1)),
```

Comment:
what you're doing here is really weird with the testing of the string. You're basically relying on something that's not necessarily documented. Things could fail if formatting changes or someone messes with streams..?

Reply:
I don't understand, I'm not testing the string.